src/scripts.js

```diff
@@ -67,6 +68,12 @@ function cardButtonConditionals(event) {
     recipeToCookCard(event);
   }
 }
+
```

Comment:
Everything looks great! My only suggestion would be to make this a strictly equals instead of soft equals

Reply:
Good catch! I had to make it strictly equal 13 (number data type not string) to make it work, but it's better this way. Nice idea! I'll push the change.